docs/docs/how-to/cloud/enable-image-cdn.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Enable Image CDN"
+description: "Learn how to accelerate builds locally and in Gatsby Cloud by offloading image processing to Gatsby's Image CDN"
+---
+
+Before continuing, make sure you understand [What is Image CDN](../../../reference/cloud/image-cdn/). Also, note that Image CDN only works on sites hosted on Gatsby cloud and doesn't work with other [hosting integrations](https://support.gatsbyjs.com/hc/en-us/articles/1500000907821-Hosting-and-Data-Source-Integrations).
+
+## 1. Enable Image CDN
+
+Image CDN is enabled per site on Gatsby Cloud. Go to Site Settings → Images → Enable Image CDN.
+
+![How to enable Gatsby's Image CDN](../../images/image-cdn-enable.png)
+
+## 2. Upgrade packages
+
+Image CDN requires the latest Gatsby 4 version. Upgrade core dependencies:
+
+```shell
+npm install gatsby@latest gatsby-plugin-image@latest gatsby-plugin-sharp@latest gatsby-transformer-sharp@latest
+```
+
+ or
+
+```shell
+yarn add gatsby@latest gatsby-plugin-image@latest gatsby-plugin-sharp@latest gatsby-transformer-sharp@latest
+```
+(note that on later npm versions you may need to install with the `--legacy-peer-deps` npm flag)
```

Comment:
````suggestion
```shell
npm install gatsby-plugin-image gatsby-plugin-sharp gatsby-transformer-sharp
```
````

1. We only ever show npm instructions, not yarn
2. `gatsby` is a given
3. `@latest` is automatically installed
4. We don't mention the legacy-peer-deps flag

But this block of information needs to be rephrased and moved completely anyways, this needs to be in the "Prerequisites" step. It would say:

You have to have these packages installed (if not done yet, here's how you install them).

At the moment the document talks about "upgrading" dependencies, that's wrong as the How-To guides always assume things with the Prerequisites.